purchases/src/main/kotlin/com/revenuecat/purchases/PresentedOfferingContext.kt

```diff
@@ -17,4 +17,23 @@ data class PresentedOfferingContext(
      * The identifier of the placement used to obtain this object.
      */
     val placementIdentifier: String? = null,
-) : Parcelable
+
+    /**
+     * The targeting context used to obtain this object.
+     */
+    val targetingContext: TargetingContext? = null,
+) : Parcelable {
+
+    @Parcelize
+    data class TargetingContext(
```

Comment:
I actually tried doing that (because it does seem better for us and the end user) but it ended up getting really messy 😬 I'll try doing it again though!